stl/inc/__msvc_int128.hpp

```diff
@@ -9,12 +9,25 @@
 
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
-#ifdef __cpp_lib_concepts
+#include <cstdint>
+#include <intrin.h> // TRANSITION, GH-2520
+#include <limits>
+#include <type_traits>
+
+#if _HAS_CXX20
 #include <bit>
 #include <compare>
+#define _ZERO_OR_NO_INIT
+#else // ^^^ _HAS_CXX20 / vvv !_HAS_CXX20
+#define _ZERO_OR_NO_INIT = {0}

```

Comment:
Could we get a comment explaining why?

Reply:
> Could we get a comment explaining why?

In case anyone is curious about the reason: uninitialized variables aren't allowed in `constexpr` functions before C++20, even if the variables are eventually written to before being read.